kodiak/main.py

```diff
@@ -83,11 +89,10 @@
 
 
 @webhook()
-async def check_run(check_run_event: events.CheckRunEvent) -> None:
+async def check_run(check_run_event: CheckRunEvent) -> None:
     """
     Trigger evaluation of all PRs included in check run.
     """
-    assert check_run_event.installation
```

Comment:
we don't need any of these assertions on installation anymore because it's not optional in the schema now.